Resources/doc/getting_started/installation.rst

```diff
@@ -12,7 +12,7 @@ following command to download the latest stable version of this bundle:
 
 .. code-block:: bash
 
-    $ composer require sonata-project/admin-bundle "2.3.*"
+    $ composer require sonata-project/admin-bundle "3.0.*"
```

Comment:
Yes, but we do not have the same version management as Symfony.

Symfony does maintain minor branches, we does not.

So targeting `3.0.*` makes no sense as we regularly release minor release (currently already on 3.2).

So having something like `^3.2` make sense, and this is the default behavior of composer when you don't specify anything.